docs/apis-tools/java-client/job-worker.md

```diff
@@ -13,6 +13,18 @@ keywords: ["backpressure", "back-pressure", "back pressure"]
 
 The Java client provides a job worker that handles polling for available jobs. This allows you to focus on writing code to handle the activated jobs.
 
+:::caution REST API limitation
+
+Camunda 8.6.0 introduces job polling via REST API. However, the HTTP client that Camunda Java client 8.6.0
+uses isn't able to maintain a long-living polling connection, i.e. perform long-polling job activation when
+activating jobs larger than the maximum message size, or when additional job activation requests are received
+while the long-polling connection is still open.
```

Comment:
> [...] or when additional job activation requests are received while the long-polling connection is still open.

We might need to clarify this a bit more:
* Does this happen when multiple job activation requests come from the same client?
* Does this happen when multiple clients run in the same JVM application?
* Does this happen when multiple clients run independently as separate JVM applications?